docs/syntax-attribute-functions-native.md

```diff
@@ -0,0 +1,54 @@
+# Native Attribute for Functions
+
+## Summary
+
+This RFC proposes a function-level `@native` attribute to request native compilation for individual functions, independent of the script-level `--!native` hotcomment.
```

Comment:
I was under the impression that these were called directives, not hotcomments. They're referred to as such on Luau's website at least: https://luau-lang.org/lint#commentdirective-26

Reply:
Changed all occurrences to "comment directive".

Reply:
We do informally call them hot comments often, but yeah, directives is better as official terminology.

Reply:
Pick any one of these at random: 1) hot comments, 2) directives, 3) pragmas.